std_msgs/Bool replan
std_msgs/UInt8 type
std_msgs/UInt8 id
---
geometry_msgs/Pose objective
std_msgs/UInt8 id
std_msgs/UInt8 code
